run-integration-test/kustomize/bases/opentelemetry-collectors/kubernetes-events/opentelemetrycollector.yml

```diff
@@ -21,12 +21,31 @@ spec:
           container.id:
             enabled: false
 
+    extensions: {}
```

Comment:
note: Again, same question and note from above.

There are some either empty fields/objects set in other files and as such the same applies to them as well. I will stop pointing them out individually.

Reply:
c80df03